plugins/org.yakindu.sct.doc.user/src/user-guide/c-domain.textile

```diff
@@ -770,6 +770,109 @@ static void statechart_enact_main_region_Check(Statechart* handle)
 }
 
 
+h2(#cdom_annotations_short_identifiers). The @ShortIdentifiers annotation
```

Comment:
As the annotation is about restrictions of the resulting C identifiers we should name it 'ShortCIdentifiers'.